doc/rtd/spelling_word_list.txt

```diff
@@ -211,6 +212,7 @@ scaleway
 seedurl
 serverurl
 setup-keymap
+shellscript
```

Comment:
I don't love this as in most places this would be a legitimate misspelling, but I couldn't find any way to ignore a spelling for a single line.

Reply:
Would not be legitimate? 

I think Chad submitted a PR in the last day or two that ignored spelling for a single line.

Reply:
> Not sure that I understand, did you mean to say "... would not be a legitimate mispelling"?

No. I think that most of the time we want "shell script" and so I think adding this has the potential to silence legitimate misspellings.